src/vcCamera.cpp

```diff
@@ -629,7 +629,7 @@ void vcCamera_HandleSceneInput(vcState *pProgramState, udDouble3 oscMove, udFloa
   cameraSurfacePosition += pProgramState->camera.cameraUp * CameraGroundBufferDistanceMeters;
 
   // Calculate if camera is underneath earth surface
-  pProgramState->camera.cameraIsUnderSurface = udDot3(pProgramState->camera.cameraUp, udNormalize3(pProgramState->camera.position - cameraSurfacePosition)) < 0;
+  pProgramState->camera.cameraIsUnderSurface = pProgramState->geozone.srid == 0 ? false : udDot3(pProgramState->camera.cameraUp, udNormalize3(pProgramState->camera.position - cameraSurfacePosition)) < 0;
```

Comment:
Check for `pProgramState->geozone.projection == udGZPT_Unknown` instead of `srid == 0`

Reply:
Also you could structure this a bit nicer:
```
pProgramState->camera.cameraIsUnderSurface = (pProgramState->geozone.projection != udGZPT_Unknown) && (udDot3(pProgramState->camera.cameraUp, udNormalize3(pProgramState->camera.position - cameraSurfacePosition)) < 0);
```